connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -287,6 +287,7 @@ protected void initializeAndStart() {
 
         if (SinkConnectorConfig.hasTopicsConfig(taskConfig)) {
             String[] topics = taskConfig.get(SinkTask.TOPICS_CONFIG).split(",");
+            Arrays.parallelSetAll(topics, i->topics[i].trim());
```

Comment:
do we really need to process this in parallel? seems like `setAll()` should be simple and clean enough.

Reply:
yeah, that's a good point. I have modified the code per your suggestion.